external/common.cmake

```diff
@@ -0,0 +1,69 @@
+# -----------------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2016, Numenta, Inc.  Unless you have purchased from
+# Numenta, Inc. a separate commercial license for this software code, the
```

Comment:
nit, to new files you don't have to add the numenta copyright, unless you want to ofc.

Reply:
>  you don't have to add the numenta copyright, unless you want to ofc.

Everything I do is donated to Numenta.  I have no need to retain any rights.